tests/zkevm/test_worst_memory.py

```diff
@@ -49,37 +50,71 @@ class CallDataOrigin:
         pytest.param(1024 * 1024, id="1MiB"),
     ],
 )
-@pytest.mark.parametrize("fixed_dst", [True, False])
+@pytest.mark.parametrize(
+    "fixed_src_dst",
+    [
+        True,
+        False,
+    ],
+)
```

Comment:
I made tha non-fixed variant imply non-fixed both in src and dst at the same time. While it can be debatable that might worth separating in dimensions, I think this provides good coverage without blowing up this (and all other tests below) by 2x.

I'm starting to get worried about being able to prove all the benchmark tests in zkVMs already. I prefer to leave it this way until I can do a full run of the whole benchmark suite doing real proving (not only counting cycles), and we can see if there's variability in some cases to justify more fine grained explorations.

For you to have an idea: the fastest block real proving time in our prover is around 4m. But is normal that many blocks take 15m, 30m or in worst cases hours.

Reply:
Interesting results! This makes me curious what the difference between those blocks are (because a 15x diff in time is very big). While I'm still treating this all as a black box this also makes me excited to start studying this a bit more.

Reply:
I'll share soon with you some preliminary results about these tests run in provers :)